package.json

```diff
@@ -5,6 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "mocha --recursive ./api/test/**/*Test.js",
+    "start": "NODE_ENV=development nodemon server.js",
```

Comment:
This may not work on all systems. If you can't start your server locally, try `NODE_ENV=development npm start`.